README.md

```diff
@@ -1,3 +1,29 @@
-# Paper Extension
-A chrome extension to sync the highlighted text from web to a dropbox paper document. 
+<h1  align='center'>
+Paper extension
+</h1> 
+<p align='center'>
+<img src='./img/icon-large.png' >
+</p>
+<h5 align='center'>
+Take notes without leaving the tab 
+</h5> 
 
+------------
+
+### Download
+- Download the .zip file of extension using the green coloured *Clone or download*  button.
+- Extract its content at the desired location.
+
+### Authorize
+- Get your *client_id* by following the instructions [here](http:/https://auth0.com/docs/connections/social/dropbox/ "here") and fill in the "*enter client id*" placeholder in paper.js.
```

Comment:
Please correct the URL, it should be: `[here](https://auth0.com/docs/connections/social/dropbox/)`